include/openthread/platform/crypto.h

```diff
@@ -170,8 +170,9 @@ typedef struct otPlatCryptoSha256Hash otPlatCryptoSha256Hash;
  */
 typedef struct otPlatCryptoEcdsaKeyPair
 {
-    uint8_t mDerBytes[OT_CRYPTO_ECDSA_MAX_DER_SIZE];
-    uint8_t mDerLength;
+    uint32_t mKeyRef;
```

Comment:
Per earlier comment, suggest keeping this as before, `mDerBytes[OT_CRYPTO_ECDSA_MAX_DER_SIZE];`.
Which can remain unused in case `mKeyRef` is being used.

Reply:
Changing back the implementation to make the array available in both implementations.